Hummm, there is a bunch of warnings now after the last rebasing:

```
/home/travis/build/python/cpython/Parser/pegen/pegen.h:194:59: warning: incompatible pointer to integer conversion passing 'char [45]' to parameter of type 'int' [-Wint-conversion]
        return _PyPegen_raise_error(p, PyExc_SyntaxError, "%s only supported in Python 3.%i and greater",
                                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

@isidentical could you take a look?

--- a/Parser/pegen/pegen.h
+++ b/Parser/pegen/pegen.h
@@ -191,8 +191,8 @@ INVALID_VERSION_CHECK(Parser *p, int version, char *msg, void *node)
     }
     if (p->feature_version < version) {
         p->error_indicator = 1;
-        return _PyPegen_raise_error(p, PyExc_SyntaxError, "%s only supported in Python 3.%i and greater",
-                                    msg, version);
+        return RAISE_SYNTAX_ERROR("%s only supported in Python 3.%i and greater",
+                                  msg, version);
     }
     return node;
 }

